packages/sha256-universal/src/node_hash.ts

```diff
@@ -0,0 +1,24 @@
+import { Checksum, SourceData } from "@aws-sdk/types";
+import { Hash as SdkNodeHash } from "@aws-sdk/hash-node";
+
+// TODO: remove the following when SdkNodeHash has implemented Checksum interface in aws-sdk-js-v3
+export class NodeHash implements Checksum {
```

Comment:
There is circular dependency between the `aws-sdk-js-crypto-helpers` and `aws-js-sdk-v3`. 

`@aws-sdk/hash-node`'s Hash implementation needs to be updated to use Checksum interface after this PR is merged, then this class will also be removed afterwards.